README.md

```diff
@@ -221,8 +221,8 @@ Check the list of pipelines you have created using the CLI:
 ```
 $ tkn pipeline ls
 
-NAME                       AGE              LAST RUN   STARTED   DURATION   STATUS
-petclinic-deploy-pipeline  25 seconds ago   ---        ---       ---        ---
+NAME     AGE              LAST RUN   STARTED   DURATION   STATUS
+voting   25 seconds ago   ---        ---       ---        ---
```

Comment:
Might be worth naming the pipeline after what it does. That said, do people create build pipelines that build and deploy two different services (voting and ui) at the same time? Usually the CI flow observes a single Git repo and build and deploys it.

@chmouel @sthaha @vdemeester thoughts?